apps/aehttp/src/aehttp_app.erl

```diff
@@ -80,6 +80,7 @@ check_env() ->
                       <<"name_service">>  => Default0,
                       <<"channel">>       => Default0,
                       <<"node_info">>     => Default0,
+                      <<"hyperchain">>    => Default0,
```

Comment:
This must be configured only when running aeternity as a child chain, not when parent chain!